bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/DropDownImplTest.java

```diff
@@ -61,6 +61,9 @@ public class DropDownImplTest {
 
     private static final String PATH_DROPDOWN_WITH_DUPLICATE_ENUMS = CONTENT_ROOT + "/dropdown-duplicate-enum";
     private static final String PATH_DROPDOWN_FOR_INSERTION_ORDER = CONTENT_ROOT + "/dropdown-insertion-order";
+    private static final String PATH_DROPDOWN_FOR_NO_ENUM_NAMES = CONTENT_ROOT + "/dropdown-without-enumNames";
```

Comment:
I could not see any test content with no enum names. Is it a miss?

Reply:
@nit23uec This was intentional, I had a test case for this [(see here)](https://github.com/adobe/aem-core-forms-components/blob/a302c74e46954b719a66578179e6c8b7da3cce22/bundles/af-core/src/test/resources/form/dropdown/test-content.json#L215) but had to remove it since supporting it would mean adding non backward compatible changes.
There are already test cases that are present for other components (checkbox, TnC) that require enumNames to remain empty even when enums are given. [(Ref)](https://github.com/adobe/aem-core-forms-components/blob/395f3069fbf9d4b6541a2f624e27dacf31b9c79b/bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/CheckBoxImplTest.java#L313)

The only solution left is to unblock enums from being returned when enumNames is empty which is what the current code does.

Reply:
@nit23uec I had [a test case](https://github.com/adobe/aem-core-forms-components/blob/a302c74e46954b719a66578179e6c8b7da3cce22/bundles/af-core/src/test/resources/form/dropdown/test-content.json#L215) but removed it intermittently, since I saw there could be backward incompatible changes. Came up with an alternative fix that evades that route. Added the test case back.